release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -59,6 +59,20 @@ This release adds improvements related to the following components and concepts.
 [id="ocp-4-13-oc"]
 === OpenShift CLI (oc)
 
+[id="ocp-4-12-import-image"]
+==== --import-mode enhancement for OpenShift CLI (oc)
+
+As of {product-title} 4.12, limited usability (Technology Preview) support for manifest listed images on image streams is available.
+
+With {product-title} {product-version}, a new `oc` command line interface (CLI) flag, `--import-mode`, has been added to the following `oc` commands:
+
+* `oc import-image`
+* `oc tag`
+
+With this enhancement, users can set the `--import-mode` flag to `Legacy` or `PreserveOriginal`, which provides users the option to import a single sub-manifest, or all manifests, of a manifest list when running the aforementioned `oc` commands.
```

Comment:
```suggestion
With this enhancement, users can set the `--import-mode` flag to `Legacy` or `PreserveOriginal`, which provides users the option to import a single sub-manifest, or all manifests, of a manifest list when running the new`oc` commands.
```

Reply:
I would either cut it entirely or maybe go with new for minimalism.

Reply:
Conflicted here. Those aren't the new commands. The --import-mode is an enhancement _to_ those commands, and has to be run with them.